forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -6006,7 +6006,7 @@ protected final boolean canBeEnchantedBy(final Card aura) {
 
     @Override
     protected final boolean canBeEquippedBy(final Card equip) {
-        return isCreature() && isInPlay();
+        return (isCreature() || isPlaneswalker()) && isInPlay();
```

Comment:
Pretty sure this would break SBA check and keep normal equipment illegally attached on a Gideon PW

You may have to validate this case stricter with the Keywords

Reply:
yeah we probably need some rules update how that should work

Reply:
See what you think of the new version